cmd/csi-sanity/main.go

```diff
@@ -81,6 +81,8 @@ func main() {
 	stringVar(&config.RemoveTargetPathCmd, "removemountpathcmd", "Command to run for target path removal")
 	stringVar(&config.RemoveStagingPathCmd, "removestagingpathcmd", "Command to run for staging path removal")
 	durationVar(&config.RemovePathCmdTimeout, "removepathcmdtimeout", "Timeout for the commands to remove target and staging paths, in seconds")
+	stringVar(&config.CheckPathCmd, "checkpathcmd", "Command to run to check a given path")
```

Comment:
It's a bit hard for users of csi-sanity to find out what that command needs to do. Right now they have to dig into the source code to find the command in `pkg/sanity/sanity.go`.

Perhaps add `It must print "file", "directory", "not_found", "other" on stdout.`?

Reply:
Done